### ISSUE TITLE
Rotate: grid not working

### ISSUE THREAD

**Issue by magnunor** [hyperspy/hyperspyUI#100]
On Ubuntu 16.04, using master branch of HyperSpyUI the optional grid in the rotate window does not do anything: no grid shows up:

```Python
s = hs.signals.Signal2D(np.random.random(size=100, 100)))
s.plot()
```
Then press "Rotate" button on sidebar (opens the rotate window), then tick preview in the rotate window (gives an interactive preview window), tick Grid (nothing happens).

**Comment by vidartf** [comment]
Yes, the grid is one big TODO. Good to have an issue to track this :)
